service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/prepopulator/section/HearingDetailsSectionPrePopulator.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.fpl.service.orders.prepopulator.section;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.order.Order;
+import uk.gov.hmcts.reform.fpl.model.order.OrderSection;
+
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.fpl.model.order.OrderSection.HEARING_DETAILS;
+
+@Component
+public class HearingDetailsSectionPrePopulator implements OrderSectionPrePopulator {
+
+    @Override
+    public OrderSection accept() {
+        return HEARING_DETAILS;
+    }
+
+    @Override
+    public Map<String, Object> prePopulate(CaseData caseData) {
+        Order type = caseData.getManageOrdersEventData().getManageOrdersType();
+        // REFACTOR: 12/04/2021 remove and replace with field interpolation once EUI-3581 is complete
```

Comment:
Apparently this is resolved, would it be worth giving it a go here?

Reply:
I think that might be savvy to keep it there, since we can have more control over the text that is captured, instead of tightening to the UI logic

Reply:
if so do you want to remove the refactor comment?

Reply:
👍